Cargo.toml

```diff
@@ -64,7 +64,7 @@ actix2 = ["actix-base", "paperclip-actix/actix2"]
 actix3 = ["actix-base", "paperclip-actix/actix3"]
 actix-base = ["v2", "paperclip-macros/actix"]
 
-# OpenAPI support (v2 and codegen)
```

Comment:
This line has been removed. You probably meant to move the feature on top of this line?